README.md

```diff
@@ -8,6 +8,36 @@ Gluon is a firmware framework to build preconfigured OpenWrt images for public m
 
 ## Getting started
 
+Gluon provides an easy-to-use firmware for a public, decentral WLAN and/or wire based mesh network.
+Which common network capable devices, like smartphones, laptops or desktop PCs can connect to and communicate over, without the need of passwords for access and without the need of installing special software.
+Additionally, internet access and "cloud merging" can be provided over a WAN through VPN connected gateways.
```

Comment:
1. Grammarly suggests to use a comma after Additionally - so I would stay with that?
2. cloud merging came in by @T-X  and means, that different mesh clouds are merged - i.e. that you can access nodes from one mesh cloud (aka domain/segment) from another one in your community. If you have a better term for that, I am open to suggestions

Reply:
1. interesting http://grammartips.homestead.com/adverbs2.html

2. My mind connects "cloud" to a server infrastructure or cloud storage but not to a mesh cloud.
```suggestion
Additionally, internet access and merging of mesh clouds can be provided over a WAN through VPN connected gateways.
```

Reply:
The text could also link to:
https://www.open-mesh.org/projects/batman-adv/wiki/Connecting-Batman-adv-clouds

Reply:
Sorry for the late response. I like using the term `merging of mesh clouds` instead of `"cloud merging"`.

However the link is wrong / completely misleading. That discusses a different topic / idea we had ages ago and was never implemented in batman-adv. The thought experiment back then was to instead of merging batman-adv clouds over a VPN, to interconnect multiple clouds on layer 3. And how to dynamically optimize the gateway selection and metric exchange between batman-adv and layer 3. And that's definitely not supported like that in Gluon right now.

Reply:
thank you for fixing my mistake!